src/main/java/ch/njol/skript/expressions/ExprColorOf.java

```diff
@@ -116,103 +129,155 @@ public Class<?>[] acceptChange(ChangeMode mode) {
 		if (mode != ChangeMode.SET && !getExpr().isSingle())
 			return null;
 
-		if (Entity.class.isAssignableFrom(returnType))
-			return CollectionUtils.array(Color.class);
-		else if (Block.class.isAssignableFrom(returnType))
-			return CollectionUtils.array(Color.class);
-		if (ItemType.class.isAssignableFrom(returnType))
-			return CollectionUtils.array(Color.class);
-		return null;
+		if (mode == ChangeMode.ADD || mode == ChangeMode.REMOVE)
+			return null;
```

Comment:
I think that if they want to add a color to the chestplate they should just use ExprDyed, adding a color to a variable sounds like it would add it as a separate element

Reply:
that's fair

Reply:
thought about `[:partially] dye %itemtype% %color%`?